geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloFeatureWriterTest.scala

```diff
@@ -434,7 +434,7 @@ class AccumuloFeatureWriterTest extends Specification with TestWithDataStore wit
       // ensure that the second part of the UUID is random
       rowKeys.map(_.substring(19)).toSet must haveLength(5)
 
-      val ids = rows.map(e => serializer.deserialize(e.getValue.get).getID)
+      val ids = rows.map(e => RecordTable.getIdFromRow(sft)(e.getKey.getRow))
```

Comment:
Are there other uses of getIdFromRow in the code base?  I.e., is there more clean-up which could be done around this?